manifests/yum/repo/internet_simp_dependencies.pp

```diff
@@ -0,0 +1,33 @@
+# Configure yum to use the internet public repository for SIMP dependencies
+#
+# @param simp_release_slug
+#   The unique release "slug" of SIMP for the target release
```

Comment:
The documentation here should call this a "URL slug" so its definition is easier to find.